source/get-started/bare-metal-install-server.rst

```diff
@@ -335,6 +338,7 @@ swap partition
 
 #. Press :kbd:`Enter`.
 
+
```

Comment:
Was this extra blank line intentional?

Reply:
Not intentional. I removed it anyway. It has zero impact on doc builds and CI/CD.